client/src/components/Overlay.md

```diff
@@ -0,0 +1,2 @@
+```js
```

Comment:
I think you forgot to add the component to the ```js part :)

Reply:
This is not part of the review.